lib/table_sticky_headers.dart

```diff
@@ -202,10 +226,24 @@ class _StickyHeadersTableState extends State<StickyHeadersTable> {
 class CellDimensions {
   const CellDimensions({
     /// Content cell width. It also applied to sticky row width.
-    @required this.contentCellWidth,
+    this.contentCellWidth,
```

Comment:
Having user enter OR parameters seems confusing. 
Unfortunately for backward compatibility your code is correct. But for ease of use two factory constructors may be created and normal constructor marked as deprecated.